modules/1_prepare/prepare.tf

```diff
@@ -322,7 +327,7 @@ resource "null_resource" "bastion_packages" {
   provisioner "remote-exec" {
     inline = [
       "#sudo yum update -y --skip-broken",
-      "sudo yum install -y wget jq git net-tools vim python3 tar"
+      "sudo yum install -y wget jq git net-tools vim python3 tar grub2-tools-extra"
```

Comment:
What is the purpose of this package with context to this PR? Suggest another PR or commit with the details.

Reply:
this is to install grub2-mknetdir, which ocp4-helpernode will use.

Reply:
This is required for CentOS9/RHEL9.